service/app/controllers/coursemanagement/validator/CourseBatchRequestValidator.java

```diff
@@ -118,18 +111,6 @@ public void validateUserId(Request courseRequest) {
     }
   }
 
-  private void validateParticipants(Request request) {
-    if (request.getRequest().containsKey(JsonKey.PARTICIPANTS)
-        && !(request.getRequest().get(JsonKey.PARTICIPANTS) instanceof List)) {
-      throw new ProjectCommonException(
-          ResponseCode.dataTypeError.getErrorCode(),
-          ResponseCode.dataTypeError.getErrorMessage(),
-          ResponseCode.CLIENT_ERROR.getResponseCode(),
-          JsonKey.PARTICIPANTS,
-          "Array");
-    }
-  }
```

Comment:
are you sure we should get rid of validate participants? aren't this validation needed for add user to batch api? please confirm.

Reply:
there is no validation for participants needed for add user to batch api